service-binding/lambda/deployment/lambda-function.yaml

```diff
@@ -64,3 +64,16 @@ spec:
  usedBy:
    kind: function
    name: redis-client
+---
+apiVersion: gateway.kyma-project.io/v1alpha2
+kind: Api
+metadata:
+  labels:
+    function: redis-client
+    example: service-binding-lambda
+  name: service-binding-lambda
+spec:
+  hostname: redis-client-test
```

Comment:
The curl command is calling redis-client.xxx, so I guess the hostname needs to be adjusted to it?

Reply:
nope, it will get the domain auto assigned.